api/test_helpers.go

```diff
@@ -2,98 +2,167 @@ package api
 
 import (
 	"encoding/json"
-	"path/filepath"
+	"path"
 	"testing"
 
-	"github.com/ethereum/go-ethereum/accounts/keystore"
-	"github.com/status-im/status-go/accounts-management/keystore/geth"
+	"github.com/status-im/status-go/accounts-management/common"
+	accscommon "github.com/status-im/status-go/accounts-management/common"
+	"github.com/status-im/status-go/accounts-management/generator"
 	"github.com/status-im/status-go/eth-node/types"
 	"github.com/status-im/status-go/multiaccounts"
 	"github.com/status-im/status-go/multiaccounts/accounts"
 	"github.com/status-im/status-go/multiaccounts/settings"
 	"github.com/status-im/status-go/params"
+	"github.com/status-im/status-go/protocol/encryption/multidevice"
 	"github.com/status-im/status-go/protocol/requests"
 	"github.com/status-im/status-go/protocol/tt"
 
 	"github.com/stretchr/testify/require"
 )
 
-func setupWalletTest(t *testing.T, password string) (backend *GethStatusBackend, defersFunc func(), err error) {
+const (
+	testPassword = "test-password"
+)
+
+var (
+	networks = json.RawMessage("{}")
+)
+
+type setupContext struct {
+	backend        *GethStatusBackend
+	mnemonic       string
+	settings       settings.Settings
```

Comment:
```suggestion
	settings       *settings.Settings
```

Reply:
Will be changed in the follow-up PR, here https://github.com/status-im/status-go/pull/6777